tests/util/kube_utils.go

```diff
@@ -597,3 +599,13 @@ func CheckDeploymentsReady(ns string) (int, error) {
 	}
 	return notReady, nil
 }
+
+// GetKubeConfig will create a kubeconfig file based on the active environment the test is run in
+func GetKubeConfig(filename string) error {
+	_, err := ShellMuteOutput("kubectl config view --raw=true --minify=true > %s", filename)
+	if err != nil {
+		return err
+	}
+	log.Infof("kubeconfig file %s created\n", filename)
+	return nil
+}
```

Comment:
Now that multiple clusters are supported, the functions in this module may be organized with methods (instead of functions). This may require to create a kubectl object that's associated with each cluster. 

A similar example is the appManager.